### PR TITLE
kat sitter overlay

### PR DESCRIPTION
**Pull request by romangraef**
this is a PR that actually works for once

### CODE REVIEW

**Review by Lulonaut — DISMISSED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java [hunk 1]
```diff
@@ -1470,4 +1476,15 @@ public static boolean isWithinRect(int x, int y, int left, int top, int width, i
         return left <= x && x <= left + width &&
                 top <= y && y <= top + height;
     }
+
+    public static <T, U> List<U> runningReduce(Iterable<T> values, U initial, BiFunction<T, U, U> reducer) {
```
Comment (ThatGravyBoat) [inline src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:1480]: Why is this here? it is not used anywhere but was added in the PR?

Comment (romangraef) [reply src/main/java/io/github/moulberry/notenoughupdates/util/Utils.java:1480]: Right, i used that in an earlier iteration of the pet xp calculation, will remove.

**Review by ThatGravyBoat — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]
